### DISCUSSION TITLE
Rooftops: green(rooftop start) -> green -> green -> green -> red (marks location)

### DISCUSSION THREAD

**Discussion by SkrillexAkaCraft** [runelite/runelite#15065] [Ideas]
so i am doing canifis rooftops and happened to find a better way to declare marks drops, so for now every time i get a marks on the 3rd obstacle everything infront is all marked orange except for the start of the rooftop being green.

what i was thinking is, why not make the obstacles infront of the marks green/orange(like it is now) and the obstacle which you can click on after the marks red or disabled so you are forced to pickup the marks before progressing the rooftop course?

my suggestion is if a marks is dropped on the 4th obstacle that the rooftop would look like this:

green(rooftop start) -> green -> green -> green -> red (marks location)

**Comment by deathbeam** [comment]
How does having the orange overlays all the time not force you to pickup the mark? That is quite big warning imo.

**Comment by SkrillexAkaCraft** [comment]
well i mean making the obstacle on the rooftop where the marks is located at red is a better warning than making it orange since everything infront of the rooftop where the marks is located is orange aswell

**Comment by RickertdeH** [comment]
I understand what you mean. I guess there could be a couple options for the users to choose from.

1.4.0: orange(rooftop start) -> orange -> orange -> orange -> orange (marks location)
Your suggestion: green(rooftop start) -> green -> green -> green -> red (marks location)
My suggestion: orange(rooftop start) -> orange -> orange -> orange -> red (marks location)

It would be cool if people can choose this so they can have whatever fits them best.
